Dockerfile

```diff
@@ -61,6 +77,7 @@ RUN apt-get update && \
 
 COPY nginx-proxy.conf /etc/nginx-proxy.conf
 COPY supervisord.conf /etc/supervisord.conf
+RUN ln -s /usr/local/openjdk-8/bin/java /usr/bin/java
```

Comment:
java path has been change from `/usr/bin/java` to `/usr/local/openjdk-8/bin/java`